clang/test/Driver/sycl-optimize-framework.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: %clangxx -fsycl -O0 -### %s 2>&1 | FileCheck %s -check-prefix=NO-OPT-CHECK
+
+// NO-OPT-CHECK-NOT: fsycl-optimize-framework
+
+// RUN: %clangxx -fsycl -O0 -fsycl-optimize-framework -### %s 2>&1 | FileCheck %s -check-prefix=OPT-CHECK
```

Comment:
I would assume this should also work for Windows?  We should verify that the behavior is correct with `%clang_cl -Od`

Reply:
Added corresponding tests.